build/Nuget/Microsoft.NET.Nuget.proj

```diff
@@ -22,32 +28,26 @@
     <PackageAssets Include="$(OutDir)**\*.*" Exclude="*.log" />
   </ItemGroup>
 
-  <Target Name="Build"
+    <!-- This project is being used as a wrapper project for packing with the nuspec.
+         https://github.com/NuGet/Home/wiki/Adding-nuget-pack-as-a-msbuild-target#packing-using-a-nuspec
+    -->
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <NuspecFile>Microsoft.NET.Sdk.nuspec</NuspecFile>
+    <NuspecBasePath>$(RootOutDir.TrimEnd('\'))</NuspecBasePath>
+    <NuspecProperties>version=$(Version)</NuspecProperties>
+    <PackageOutputPath>$(NuGetOutDir.TrimEnd('\'))</PackageOutputPath>
```

Comment:
I had to set this for it to be dumped inside Packages instead of in NuspecBasePath.